### DISCUSSION TITLE
[BUG] Anycubic Mono X 6k - PWMB Problems

### DISCUSSION THREAD

**Discussion by antwal** [sn4k3/UVtools#539] [Q&A]
## System
 - OS: macOS Catalina
 - Slicer: UVTools
 - Printer: Anycubic Mono X 6K
 - UVtools version: 3.5.6

```
UVtools v3.5.6 X64
Operative system: Darwin 19.6.0 Darwin Kernel Version 19.6.0: Mon Apr 18 21:50:40 PDT 2022; root:xnu-6153.141.62~1/RELEASE_X86_64 X64
Processor: Intel(R) Core(TM) i5-4570S CPU @ 2.90GHz
Processor cores: 4
Memory RAM: 8.98 / 14.58 GB
Runtime: osx.10.15-x64
Framework: .NET 6.0.6
AvaloniaUI: 0.10.17
OpenCV: 4.5.4-235

Sreens, resolution, working area, usable area:
1: 1920 x 1080 @ 100% (Primary)
    WA: 1920 x 1057    UA: 1920 x 1057
2: 1366 x 768 @ 100%
    WA: 1366 x 745    UA: 1366 x 745

Path:       /Applications/UVtools.app/Contents/MacOS/
Executable: /Applications/UVtools.app/Contents/MacOS/UVtools
Loaded file: UVtools_demo_file.sl1 [Version: 0] [Class: SL1File]

```

## Describe the bug
a size error occurs during the file conversion then generates this problem on the printout (see screenshot)

## To Reproduce
1. Open UVTools
2. Menu Calibration -> Elephant foot
3. File -> Convert to -> Photon Mono 6K
4. Version 1 or 515 or 516

## Screenshots
<img width="1210" alt="Schermata 2022-08-05 alle 21 15 56" src="https://user-images.githubusercontent.com/410736/183145845-0a9053eb-0f2a-4d94-9536-71a5d47d8a30.png">

![20220804_162311_1](https://user-images.githubusercontent.com/410736/183145898-b4b61a5c-eda4-43cb-86a5-f9019509fa8d.jpg)

![20220804_162305_1](https://user-images.githubusercontent.com/410736/183145909-0497c544-caf1-4c15-a6d7-e18f4daa55ba.jpg)

**Comment by sn4k3** [comment]
You can't use the demo file to generate tests or whatever to your printer.
You must load and use a dummy previous sliced file by you for your printer.

Demo file is just there to able to explore UVtools without a file but you ALWAYS need to open your own based on your printer.
Printer firmware is dumb enough to accept such malformed file...

**Comment by antwal** [comment]
I understand, but I also have other problems with the files generated on UVTools, I am attaching the problems and PWMB files for possible debugging

Slicer is PrusaSlicer

[PWMB Files](https://ufile.io/f/vwtiu)

Version 1 and 515 the Z axis does not go up

With version 516 it works but the prints fail, I also attach the screenshots of the printed pieces.

![photo_2022-08-11_23-30-14](https://user-images.githubusercontent.com/410736/184359044-355b0fba-65f8-4d34-9618-5749adad5f79.jpg)

![photo_2022-08-11_23-30-13_1](https://user-images.githubusercontent.com/410736/184359062-424fd5d0-f187-47d4-922e-7a086247e9a0.jpg)

![photo_2022-08-11_23-30-13](https://user-images.githubusercontent.com/410736/184359076-d694bd76-9b71-4f4e-b873-203f760f75ce.jpg)

**Comment by sn4k3** [comment]
> Version 1 and 515 the Z axis does not go up. With version 516 it works.

Newest Anycubic machines uses only v516, other versions will fail.
With my PS profiles it output the correct version.

> but the prints fail

Can't see where this is realated with UVtools...
Either your supports are failling or you are one of the unlucky guys to get the machine with bad board that shift image when AA is ON. Try to print same file but with AA disabled (You must slice on Photonworkshop and disable AA)

**Comment by antwal** [comment]
> > Version 1 and 515 the Z axis does not go up. With version 516 it works.
> 
> Newest Anycubic machines uses only v516, other versions will fail. With my PS profiles it output the correct version.
> 

i installed the profile from uvtools it should be correct right?

> > but the prints fail
> 
> Can't see where this is realated with UVtools... Either your supports are failling or you are one of the unlucky guys to get the machine with bad board that shift image when AA is ON. Try to print same file but with AA disabled (You must slice on Photonworkshop and disable AA)

ok I also try this thing


with photon workspace 2.1.29 and lycheeslicer the printing is successful without problems, I also tried chitubox free and pro generates problems similar to prusa + uvtools

**Comment by sn4k3** [comment]
> i installed the profile from uvtools it should be correct right?

Yes

photon workspace & lycheeslicer by default dont have AA enabled. While PrusaSlicer haves and Chitubox may also have

**Comment by antwal** [comment]
ok can i disable it for testing to check if it works, on prusa? thx

**Comment by sn4k3** [comment]
On prusa you can disable AA setting gamma to 0 BUT you also need to disable on UVtools which you need to use the terminal console for that (No direct way on UI) that why i adviced to try with photon workshop.

You can always slice on prusa and export to stl and finish on photonworkshop
